README.md

```diff
@@ -1,2 +1,48 @@
 # Hello-Microverse-Project
-Day 2 Week 1 Project Of Setting Up "Hello Mivroverse" With HTML And CSS Project
+Day 2 Week 1 Project Of Setting Up "Hello Mivroverse" With HTML And CSS Project.
+
+
+
+# Built with
+<ul>
+<li>HTML</li>
+<li>CSS</li>
+</ul>
+
+# Getting Started
+The following is the instructions for creating this project. Follow to understand.
+
+# Prerequisites
+VSCode
+Git
+Browser
+ # Setup
+<p>Create a directory in Github.<br>
+Clone the directory to local repository and make it available locally.<br>
+From the main branch,create a branch eg:  file-creation branch  to create the necessary files required for the project which will be merged to the main branch. These Files include index.html and style.css.<br>
+Then create-linter-files branch  to create the linter files. Then later will be merged to the main branch.<br>
+Open VSCode, choose open folder and open the directory that was cloned from Github.<br>
+Open the index.html and populate it with boiler plate code and a h1 element with the text Hello Microverse and assign a class to it of name header
+Link the stylesheet to index.html.<br>
+Open the style.css and add the style properties to the h1 element.
+
+# Run Tests
+Run the index.html on your preferred browser or use a live server.
+
+# Authors
+David Kasilia Mwanzia
+
+# Contributing
+Contributions, issues, and feature request are welcome anytime.
+
+# Show your support
+Give a Like or a Star if you like this project.
+
+# Acknowledgements
+Microverse
+
+# Links
+<a href="https://github.com/David-Kasilia/Hello-Microverse-Project/pull/8">Pull Request From The Review and Changes Made</a>
+
+
+
```

Comment:
- [ ] It is great that you have added a readme file :+1: But you didn't add your information and your project information in the readme. You can checkout this [readme](https://github.com/HAMMAS-SALEEM/Hello-Microverse/blob/master/README.md) to get a good understanding of what am I saying. :+1: